src/hooks/ykfde

```diff
@@ -31,7 +31,7 @@ run_hook() {
   _tmp=""
   local cryptopt cryptoptions
 
-  [ -x /bin/plymouth ] && plymouth --ping && YKFDE_USE_PLYMOUTH=1
+  [ -x /bin/plymouth ] && [ $splash] && plymouth --ping && YKFDE_USE_PLYMOUTH=1
```

Comment:
Please take a look at [contribution guide](https://github.com/agherzan/yubikey-full-disk-encryption/blob/master/CONTRIBUTING.md#open-pull-request) and fix warnings like below:
```
In ykfde line 34:
  [ -x /bin/plymouth ] && [ $splash] && plymouth --ping && YKFDE_USE_PLYMOUTH=1
                          ^-- SC1073: Couldn't parse this test expression. Fix to allow more checks.
                                    ^-- SC1020: You need a space before the ].
                                    ^-- SC1072: Missing space before ]. Fix any mentioned problems and try again.

For more information:
  https://www.shellcheck.net/wiki/SC1020 -- You need a space before the ].
  https://www.shellcheck.net/wiki/SC1072 -- Missing space before ]. Fix any m...
  https://www.shellcheck.net/wiki/SC1073 -- Couldn't parse this test expressi...
```